gettsim/data/soli_st.yaml

```diff
@@ -1,43 +1,51 @@
-solifreigrenze:
+solisatz:
   name:
-    de: Soli-Freigrenze  (§ 3 (3) Nr. 2 SolzG)
+    de: Zuschlagsatz
+    en: Tax rate
+  description:
+    de: Höhe des Solidaritätszuschlags auf die Bemessungsgrundlage. § 4 SolZG in den Jahren 1991 und 1992. § 4 SolZG 1995 ab 1995.
     en:
+  values:
+    1991-01-01:
+      value: 0.0375
+      note: Artikel 1 G. v. 24.06.1991 BGBl. I S. 1318.
+    1995-01-01:
+      value: 0.075
+      note: Artikel 31 G. v. 23.06.1993 BGBl. I S. 944.
+    1998-01-01:
+      value: 0.055
+      note: Artikel 1 G. v. 21.11.1997 BGBl. I S. 2743.
+solifreigrenze:
+  name:
+    de: Freigrenze
+    en: Tax exempt
   description:
-    de:
+    de: Freigrenze beim Solidaritätszuschlag 1995. § 3 (3) Nr. 2 SolZG 1995.
     en:
   values:
-    1984-01-01:
-      value: 0
-      note:
     1995-01-01:
       value: 681
-      note: .
+      note: Artikel 31 G. v. 23.06.1993 BGBl. I S. 944. Wert in Euro. Der Betrag im Gesetz lautet 1332 DM.
     1998-01-01:
       value: 939
-      note:
+      note: Artikel 1 G. v. 21.11.1997 BGBl. I S. 2743. Wert in Euro. Der Betrag im Gesetz lautet 1836 DM.
     2002-01-01:
       value: 972
-      note:
-solisatz:
+      note: G. v. 15.10.2002 BGBl. I S. 4131.
+    2021-01-01:
+      value: 16956
+      note: Artikel 1 G. v. 10.12.2019 BGBl. I S. 2115.
+solimax:
   name:
-    de: Höhe des Solidaritätszuschlags auf die Einkommensteuer  (§ 4 SolzG sowie § 51a EStG)
-    en:
+    de: Prozentsatz zur Berechnung Maximalsatz
+    en: Percentage used to calculate the maximum rate
   description:
-    de:
+    de: Prozentsatz zur Berechnung des Maximalsatzes des Solidaritätszuschlags 1995. § 4 SolzG 1995. Der Solidaritätszuschlag beträgt nicht mehr als dieser Prozentanteil des Unterschiedsbetrages zwischen der festzusetzenden Einkommensteuer und der Freigrenze. Dies definiert die Phase-In-Zone.
     en:
```

Comment:
Could we call it `soli_rate_phase_in`? Because that's what it is.

Reply:
It is more complicated than that. It is not a tax rate. It is a parameter that is used to calculate the phase in rate which is less than 0.2.

Reply:
It's the marginal tax rate of the soli when you are slighty above the `solifreigrenze`. Maybe `soli_mtr_phase_in`.

Reply:
No it is not. The marginal tax rate of the soli when you are slighty above the `solifreigrenze` is approx. 4.8 percent:
> ![Bildschirmfoto von 2019-12-15 16-01-53](https://user-images.githubusercontent.com/31305679/70864753-798a2800-1f55-11ea-8f68-f20b549a773d.png)

([Source](https://www.ifo.de/DocDL/sd-2019-16-bloemer-etal-solidaritaetszuschlag-reform-2019-08-22_0.pdf#page=3))

Reply:
Correct. 20% regarding the income tax. I'm just trying to come up with a telling name.

Reply:
OK, the marginal Soli with respect to the income tax, yes. About the name: rename `solisatz` to `soli_rate` and `solimax` to `soli_rate_max`. I'm also fine with `rate` and `rate_phase_in` (but would definitely avoid `mtr').

Reply:
> About the name: rename `solisatz` to `soli_rate` and `solimax` to `soli_rate_max`?

I like.